README.md

```diff
@@ -11,9 +11,9 @@ This project contains the official Go driver for the [ArangoDB database](https:/
 > **Warning**
 > V1 is deprecated and will not receive any new features. Please use [V2](v2) instead.
 
-- [Getting Started](https://docs.arangodb.com/stable/develop/drivers/go/)
-- [Example Requests](v2/examples)
-- [Reference](https://godoc.org/github.com/arangodb/go-driver)
+- [Example Requests - V2](v2/examples)
+- [Reference - V2](https://godoc.org/github.com/arangodb/go-driver)
+- [Getting Started Tutorial - V1 only (V2 is coming soon!)](https://docs.arangodb.com/stable/develop/drivers/go/)
```

Comment:
The link to the reference docs should point to /2, right?

I'd like to add the tutorial to a Markdown file in this repo next to the v1 code and then link to that instead.

```suggestion
- [Code examples (v2)](v2/examples)
- [Reference (v2)](https://godoc.org/github.com/arangodb/go-driver/v2)
- [Tutorial (v1)](https://developer.arangodb.com/docs/category/go)
```